java/code/src/com/redhat/rhn/taskomatic/task/HubReportDbUpdateWorker.java

```diff
@@ -76,6 +79,23 @@ public void setParentQueue(TaskQueue queue) {
         this.parentQueue = queue;
     }
 
+    private List<String> filterExistingTables(Session remoteSession, Long serverId) {
+        SelectMode query = generateExistingTables(remoteSession, TABLES);
+        DataResult<Map<String, Object>> result = query.execute();
+        Set<Map.Entry<String, Object>> tableEntry = result.get(0).entrySet();
+        tableEntry.removeIf(t -> {
+            if (t.getValue() == null) {
+                log.warn("Table '" + t.getKey() + "' does not exist in session " + serverId + ": " +
+                        "this table will not be updated");
```

Comment:
... not exist in **session** ? In or on "server", right?

Reply:
right